README.md

```diff
@@ -86,91 +48,15 @@ You can override the Dapr address used by input and output bindings by setting t
-
-An example of a function app deployment for Kubernetes can be [found below](#sample-kubernetes-deployment).
-
-The following command will generate a `deploy.yaml` file for your project:
-`func kubernetes deploy --name {container-name} --registry {docker-registry} --dry-run > deploy.yaml`
-
-You can then edit the generated `deploy.yaml` to add the dapr annotations.  You can also craft a deployment manually.
-
-### Azure Storage account requirements
-
-While an Azure Storage account is required to run functions within Azure, it is NOT required for functions that run in Kubernetes or from a Docker container.  The exception to that is functions that leverage a Timer trigger or Event Hub trigger.  In those cases the storage account is used to coordinate leases for instances, so you will need to set an `AzureWebJobsStorage` connection string if using those triggers.  You can set the `AzureWebJobsStorage` value to `none` if not using any of the triggers that require it.
-
-### Sample Kubernetes deployment
-
-```yml
-apiVersion: v1
-kind: Service
-metadata:
-  name: my-function
-  namespace: default
-spec:
-  selector:
-    app: my-function
-  ports:
-  - protocol: TCP
-    port: 80
-    targetPort: 80
-  type: LoadBalancer
----
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: my-function
-  namespace: default
-  labels:
-    app: my-function
-spec:
-  replicas: 1
-  selector:
-    matchLabels:
-      app: my-function
-  template:
-    metadata:
-      labels:
-        app: my-function
-      annotations:
-        dapr.io/enabled: "true"
-        dapr.io/id: "functionapp"
-        # Only define port of Dapr triggers are included
-        dapr.io/port: "3001"
-    spec:
-      containers:
-      - name: my-function
-        image: myregistry/my-function
-        ports:
-        # Port for HTTP triggered functions
-        - containerPort: 80
----
-```
-
 ## Known Issues
 
-- **By Design:** In out-of-proc model, support for POCO model isn't available for output bindings and triggers, all the payloads must send JSON data and these data should be used as JsonElement type in Azure Functions. Please look at the [Note section][dotnet-out-of-proc] to find out required values for each bindings and triggers.
+- **By Design:** In isolated mode, there's no support for using POCO (Plain Old CLR Object) models in output bindings and triggers. All payloads must be sent as JSON data, and these data should be treated as the JsonElement type in Azure Functions. You can refer to the [input bindings][input-binding-details], [output bindings][output-binding-details], and [triggers][trigger-details] sections to understand the data format and the necessary properties for each type of binding.
+
+## Deployments
```

Comment:
Shall we still call these "quickstarts"? Users will typically look for quickstarts and 'getting started' than `deployment` when they first explore.

Reply:
I didn't get your suggestion, which part you are referring to call them "quickstarts"?

Reply:
We are providing the [quickstart](https://github.com/Azure/azure-functions-dapr-extension/pull/165/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R19) link at the start of main readme. 

There was [one query](https://github.com/Azure/azure-functions-dapr-extension/issues/88#issuecomment-1745874264) on how they deploy the Function with Dapr extension in ACA. So having this deployment section helps user to navigate when they look for deployment guidelines.